Content/posts/how-we-do-vapor-releases.md

```diff
@@ -0,0 +1,51 @@
+---
+date: 2025-08-14 18:00
+description: See how we approach releases in Vapor, so we can release quickly and often to save time and provide a better experience for users and contributors.
+tags: Framework, Automation
+author: Tim
+authorImageURL: /author-images/tim.jpg
+---
+# How we approach releases at Vapor
+
+I've had some interesting conversations recently at why we do releases the way we do in Vapor so thought it would be worth sharing. 
```

Comment:
```suggestion
I've had some interesting conversations recently about why we do releases the way we do in Vapor, and I thought it would be worth sharing some of the details.
```